README.md

```diff
@@ -67,6 +67,9 @@ First, you're going to need to copy the `.env.example` to `.env`. The current op
 | `EMAIL_USERNAME` | Gmail address; e.g. `jensen.robbed.us@gmail.com` |
 | `EMAIL_PASSWORD` | Gmail password; see below if you have MFA |
 | `STORES` | List of [stores](#Supported-stores) you want to be scraped; optional, default: `nvidia` |
+| `PHONE_NUMBER` | 10 digit phone number. ONLY FOR USA. SMS CHARGES APPLY, for example: `123456789` |
+| `CARRIER` | Carrier you have. Only supports `sprint, tmobile, att, verizon` |
+| `NOTIFICATION_METHODS` | List of methods you'd like to be notified by, default: `email` |
```

Comment:
I'm thinking that we get rid of this environment and don't have this configuration. If the user sets the value, then they should assume that it's going to be sent.

Reply:
I agree, i've removed it in this branch, should be ok now